### ISSUE TITLE
importOrderFile fails on "all unmatched type imports" group

### ISSUE THREAD

**Issue by benjaminhawkeslewis** [diffplug/spotless#2]
If you create an Eclipse import order including an "all unmatched type imports" group and attempt to use it with Spotless, Spotless errors.

Example import order file matching the [Google Java style guide](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing):

```
#Organize Import Order
#Sat Aug 22 12:43:31 BST 2015
4=javax
3=java
2=
1=com.google
0=\#
```

Note that "all unmatched type imports" (as the Eclipse UI phrases it) is represented by an empty string (number 2).

I think the error is produced in https://github.com/diffplug/spotless/blob/191e7ddc784e69cb9eea30de58d6afbe2688560d/src/main/java/com/diffplug/gradle/spotless/java/ImportSorterStep.java#L54 where it is assumed that splitting the line on `=` will produce two strings.

**Comment by nedtwigg** [comment]
Good find!  There's a test harness for the import ordering [here](https://github.com/diffplug/spotless/blob/191e7ddc784e69cb9eea30de58d6afbe2688560d/src/test/java/com/diffplug/gradle/spotless/java/ImportSorterStepTest.java), which should make it easy to test a fix.

I should be able to look into a fix by Wednesday.  If you'd like to do it yourself before then, I'd be happy to [help in the gitter chat](https://gitter.im/diffplug/spotless) and merge the PR.

**Comment by nedtwigg** [comment]
Implemented in [1.2.0](https://github.com/diffplug/spotless/releases/tag/v1.2.0).
